### PR TITLE
Operator version 3.0.2

### PR DESCRIPTION
**Pull request by rjeberhard**
We are planning to release another set of bug fixes.

The following PR's were selected based on customer impact and severity:
* #1867 
* #1869 
* #1872 
* #1878 
* #1879 

Additionally, the following PR's were required because one or more of the PR's in the above list depended on these changes:
* #1827 
* #1832 

Based on a review request, additionally added
* #1842

### CODE REVIEW

**Comment by tbarnes-us** [comment]
Can you please also consider https://github.com/oracle/weblogic-kubernetes-operator/pull/1842?  It's just a FAQ update and customers seem to need the info -- I think another customer ran into this just this week.

**Comment by rjeberhard** [comment]
Tests have passed: https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/1959/

**Review by rosemarymarano — APPROVED** [review]
From the doc perspective, LGTM.

**Comment by russgold** [comment]
What is the process of adding bug fixes? It doesn't look as though you are cherry-picking commits merged into develop, as you've been squashing the PRs to single commits, and here you seem to be using the original commits on the branch. How will this affect merging future work from develop into master?

**Comment by rjeberhard** [comment]
I cherry-picked the commits from the original PR's.

**Comment by rjeberhard** [comment]
After we merge to master, I'll merge master back into develop to make sure that there are no conflicts (there shouldn't be).

**Comment by russgold** [comment]
What is the advantage of doing that vs. cherry-picking the squashed commits?

File: docs-source/content/release-notes.md [hunk 1]
```diff
@@ -6,8 +6,9 @@ draft: false
 
 ### Recent changes
 
-| Date | Version | Introduces backward incompatibilities | Change |
+| Date | Version | Introduces backward incompatibilities? | Changes |
 | --- | --- | --- | --- |
+| September 15, 2020 | v3.0.2 | no | This release contains several fixes, including improvements to log rotation and a fix that avoids unnecessarily updating the domain status. |
```
Comment (mriccell) [inline docs-source/content/release-notes.md:11]: Can we list the fixes that went into 3.0.2 for user reference?

**Review by mriccell — CHANGES_REQUESTED** [review]

**Review by mriccell — APPROVED** [review]

File: docs-source/content/release-notes.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ date: 2019-03-15T11:25:28-04:00
 draft: false
 ---
 
-### Recent changes
+### Releases
 
 | Date | Version | Introduces backward incompatibilities? | Changes |
```
Comment (rosemarymarano) [inline docs-source/content/release-notes.md:9]: Changes -> Change (I let this go before but it was a mistake. It should be singular, change, just like all the other headings are singular, Date, Version, Issue, and Description. The question mark after Introduces backward incompatibilities? is ok.) I looked at the Change log in the Hugo UI and it all looks fine, definitely good to go.

**Review by rosemarymarano — APPROVED** [review]
One edit request/comment.

**Pull request merged**